src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -654,26 +654,26 @@ def __removeWMSTasks(self, transJobIDs):
                 self.log.error("Failed to kill jobs", "(n=%d)" % len(res["FailedJobIDs"]))
                 allRemove = False
 
-            res = self.wmsClient.removeJob(jobList)
+            res = self.wmsClient.deleteJob(jobList)
```

Comment:
Does this mean #5598 wasn't actually needed and there wasn't an issue with jobs getting stuck in deleted?

Reply:
> Does this mean #5598 wasn't actually needed and there wasn't an issue with jobs getting stuck in `JobStatus.DELETED`?

Correct, at least for productions: the good news is that contrary to what I wrote before, there's no need for manual intervention.

#5598 at the same time was applying a couple additional minor fixes.